test/perf_test/generate_perf_report.py

```diff
@@ -0,0 +1,40 @@
+# Python program to read
+# json file
+
```

Comment:
Either we create a folder under perf_test for resNet50 or have that somewhere in file-name as later if we add similar test cases for other scenarios it will be useful.

Reply:
this script will be called on all perf apps including resnet - you basically pass it a list of metrics and the two jobs to compare